audio-analysis/videoHelper.py

```diff
@@ -130,10 +137,16 @@ def getAmplitudeAnalysis(self):
                 fc = 0
                 amp_sum = 0
                 ic += 1
+
+                #Append to Video Object Amplitude List
+                self.amplitude_list.append(dict(interval_dict))
+
             fc += 1
+            fc_continuous += 1
 
-        for x, y in interval_amplitudes.items():
-            print(x, y)
+        #Commented out but decided to keep in case, loop through interval_amplitudes intervals and outputs them
```

Comment:
Maybe we can add this inside a `TESTING` if statement?

Reply:
Agreed - I'm almost tempted to just completely remove it though because the dictionary should output the same things anyways